parastell/utils.py

```diff
@@ -202,3 +207,119 @@ def smooth_matrix(matrix, steps, sigma):
         previous_matrix = smoothed_matrix
 
     return smoothed_matrix
+
+
+class DAGMCMerger:
```

Comment:
Less of a suggestion and more of a question
```suggestion
class DAGMCMerger(object):
```

Reply:
they are the same, classes inherit from object by default. I'll make the change though to avoid future confusion for anyone